oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -696,6 +759,14 @@ public Collection<String> getDefaultScopes() {
       return defaultScopes;
     }
 
+    public AuthTransport getTransport() {
+      return transport;
+    }
+
+    public BindingEnforcement getBindingEnforcement() {
+      return bindingEnforcement;
+    }
+
```

Comment:
same here. also if it's returning AuthTransport, can it be `getAuthTransport()`?

Reply:
Done in https://github.com/googleapis/google-auth-library-java/pull/1558/commits/8a2d967552c6322f7b6514dcde4b7ba190a7128a